dokka-subprojects/plugin-base-frontend/.nvmrc

```diff
@@ -0,0 +1 @@
+18.20.4
```

Comment:
I'd like it if the version wasn't duplicated both here and in the version catalog.

We could make a Gradle task that will automatically update this file to match the value in `libs.version.toml`. WDYT?

(I can do this in a new PR, to protect everyone from Gradle headaches.)

Reply:
The automatic sync between `libs.version.toml` and `.nvmrc` would be handy 👍

But it's worth noticing that this .nvmrc`  file is not very important, it's only used to make local development slightly easier
So this automation will pay off only if it's really easy and quick

Reply:
Yeah, it should just be a few lines to set up.

But if it's a dev-local thing, maybe `.nvmrc` should just be git-ignored?

Reply:
Cool then!

The `.nvmrc` helps to automatically change local node version on a developer's machine when one switches between projects, so it's convenient to have it in repo